js/main.js

```diff
@@ -1,3 +1,8 @@
+if (window.location.hash === '#more') {
```

Comment:
Not sure if we need any `$(document).ready` stuff here. Sorry for being a JS noob.